includes/html/pages/ports.inc.php

```diff
@@ -404,6 +404,11 @@
                     }
                 }
                 break;
+            case 'group':
+                $where .= ' AND port_id IN (';
+                $where .= DB::table('port_group_port')->where('port_group_id', $vars['group'])->pluck('port_id')->implode(',');
```

Comment:
Normally you do a subquery instead of two queries

Reply:
Yeah, would you cheat and just add ->toSql() on the end of the current Fluent query? :D